modules/nvidia-gpu-aws-enabling-gpu-aws-instances.adoc

```diff
@@ -0,0 +1,70 @@
+      hostPrefix: 23
+  machineNetwork:
+    - cidr: 10.0.0.0/16
+  networkType: OpenShiftSDN
+  serviceNetwork:
+    - 172.30.0.0/16
+platform:
+  aws:
+    region: us-east-1
+    userTags:
+      adminContact: jdoe
+publish: External
+pullSecret: '{"auths":{"cloud.openshift.com":{"auth":"b3BlbnNoaWZ0LXJlbGVhc2UtZGV2K2ZkdXBvbnRyZWRoYXRjb20xbnZ0eGVjZGh1Z3RzYnhrcHJuejNwa2RhdW06WVdVSzgwRE9SRUJCMDROU1c4VTRCNlI4RFlNMTNBOFlMMThTRzMyRDZYNTZBUUszMktOSTY2T0I4TjNFM0FFRg==","email":"fdupont@redhat.com"},"quay.io":{"auth":"b3BlbnNoaWZ0LXJlbGVhc2UtZGV2K2ZkdXBvbnRyZWRoYXRjb20xbnZ0eGVjZGh1Z3RzYnhrcHJuejNwa2RhdW06WVdVSzgwRE9SRUJCMDROU1c4VTRCNlI4RFlNMTNBOFlMMThTRzMyRDZYNTZBUUszMktOSTY2T0I4TjNFM0FFRg==","email":"fdupont@redhat.com"},"registry.connect.redhat.com":{"auth":"NzA3MjA1Nnx1aGMtMU5WVFhlY0RoVWdUc2J4a1ByblozcGtEYVVtOmV5SmhiR2NpT2lKU1V6VXhNaUo5LmV5SnpkV0lpT2lJek4yUXhPR1ZpT0RJeVpEUTBaV1JpT0RNNVpqRmtZbUkxTldNelpUUmxOaUo5LnFIVlpwZnZTeFpfSDB0MThqUDJuSzVFTWJ1SkNDRWhMRkx2REQyZkVydG53bHVVRGRmb2h0T05JV3pEakE2b3ROWVRfVTZRaFBoUHV4a0RrbE9UMmlNV05DaXZNdm5LejdIREg1M00xSkhZMkc5UEtUN3FVZEdXenJCMmQwVTZTMFdvbG0yTVlXQkg5QjdnbWdIUmtwZzBsMWNTS25SdUlSUEI2MFBqZ2RQWlIxZVFCYlowR1hnYlVRcVQyR1QycVc2RGNWSkVxdzEyNWIyOGhzMmNWeUhNaXhYM09MOUFNczI4b0JMeWdTb2NrQkRXb1o5MkNpek5mT01rRUhmcWZXdXJ1TDlRMzRoTjEzRVJweHBaNnNYNnZIdERrc3pJZ3Z5MFltUG9uR0JKVmMtRTlpSjZxVm9Sb3A0NVNuWnFVR2VHamFoMk1jUUhLTkt5TnRpWGJhSEVUSXZtZWd2QTg3SE1yWENMeXZHQWpMamM3ckw3UDd2aDBweVlYdU40bHdyUkRDbXZ5TnA4c2JRU3JISk13SWpyWXJqdGNHUVZIVWswX0lNdFI1MkJzdXMzZloteGFLZVJyRzFDZ3E1U0FDVlo2RjZNZE1TckZ3UnFzZndKQldWRzNxa2hQMjBpRktKS2cyV0NYQmFQMU45QnRIa1MxYUpmZUdibDktWllWeHQyUDZHaXZTdG13cnB0ZGFEeVFwQVBjZGdwYTdmT3JsY24zQ3RlazlQbWdkclluUkZNbU92bkQxSTZ2eFBrMG9GTTFXVHd4T1RTM1NKODYwY3JzT0hWb0Z3WWw4eWxxWW5DSFdlV0pKaU9peVpFa0JIZ2dVeFQxNUhTcU1IVHprbElORDRxYkV5WFlPWEE3MEd4M1ltaG5NbmEzRHdCMGhXS1hrUDNMRm9Z","email":"fdupont@redhat.com"},"registry.redhat.io":{"auth":"NzA3MjA1Nnx1aGMtMU5WVFhlY0RoVWdUc2J4a1ByblozcGtEYVVtOmV5SmhiR2NpT2lKU1V6VXhNaUo5LmV5SnpkV0lpT2lJek4yUXhPR1ZpT0RJeVpEUTBaV1JpT0RNNVpqRmtZbUkxTldNelpUUmxOaUo5LnFIVlpwZnZTeFpfSDB0MThqUDJuSzVFTWJ1SkNDRWhMRkx2REQyZkVydG53bHVVRGRmb2h0T05JV3pEakE2b3ROWVRfVTZRaFBoUHV4a0RrbE9UMmlNV05DaXZNdm5LejdIREg1M00xSkhZMkc5UEtUN3FVZEdXenJCMmQwVTZTMFdvbG0yTVlXQkg5QjdnbWdIUmtwZzBsMWNTS25SdUlSUEI2MFBqZ2RQWlIxZVFCYlowR1hnYlVRcVQyR1QycVc2RGNWSkVxdzEyNWIyOGhzMmNWeUhNaXhYM09MOUFNczI4b0JMeWdTb2NrQkRXb1o5MkNpek5mT01rRUhmcWZXdXJ1TDlRMzRoTjEzRVJweHBaNnNYNnZIdERrc3pJZ3Z5MFltUG9uR0JKVmMtRTlpSjZxVm9Sb3A0NVNuWnFVR2VHamFoMk1jUUhLTkt5TnRpWGJhSEVUSXZtZWd2QTg3SE1yWENMeXZHQWpMamM3ckw3UDd2aDBweVlYdU40bHdyUkRDbXZ5TnA4c2JRU3JISk13SWpyWXJqdGNHUVZIVWswX0lNdFI1MkJzdXMzZloteGFLZVJyRzFDZ3E1U0FDVlo2RjZNZE1TckZ3UnFzZndKQldWRzNxa2hQMjBpRktKS2cyV0NYQmFQMU45QnRIa1MxYUpmZUdibDktWllWeHQyUDZHaXZTdG13cnB0ZGFEeVFwQVBjZGdwYTdmT3JsY24zQ3RlazlQbWdkclluUkZNbU92bkQxSTZ2eFBrMG9GTTFXVHd4T1RTM1NKODYwY3JzT0hWb0Z3WWw4eWxxWW5DSFdlV0pKaU9peVpFa0JIZ2dVeFQxNUhTcU1IVHprbElORDRxYkV5WFlPWEE3MEd4M1ltaG5NbmEzRHdCMGhXS1hrUDNMRm9Z","email":"fdupont@redhat.com"}}}'
```

Comment:
remove actual pull secret

Reply:
REmoved the pullSecret and replaced with **pullSecret: '{"auths":{"cloud.openshift.com": <pullSecret>**